### ISSUE TITLE
Edition omnibar mis-aligned + clipped

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#10064]
### Problem

![image](https://github.com/user-attachments/assets/34535a64-5e4d-4be5-931d-39b38734bbc6)

Also clipped awkwardly after having scrolled:
![image](https://github.com/user-attachments/assets/2cefbfb0-ae5f-4f59-8de6-34a3cb020e46)

### Reproducing the bug

1. Go to any book page in mobile screen size, eg https://openlibrary.org/books/OL1088853M/Wizard%27s_First_Rule

* Expected behavior: The arrow is vertically centred, and the text isn't so awkwardly clipped. This should likely be full width.
* Actual behavior: Not centred; clipped.

### Context

- Browser (Chrome, Safari, Firefox, etc): FF
- OS (Windows, Mac, etc): Win
- Logged in (Y/N): Y
- Environment (prod, dev, local): prod

#### Requirements Checklist
* [ ]

#### Related files

* openlibrary/macros/EditionNavBar.html
* https://github.com/internetarchive/openlibrary/blob/77c16d530b4d5c0f33d68bead2c6b329aee9b996/openlibrary/templates/type/edition/view.html#L207

#### Stakeholders

*
<hr>

#### Instructions for Contributors

- Please [run these commands](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#working-on-your-branch) to ensure your repository is up to date **before** [creating a new branch](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-changes-and-creating-a-pull-request) to work on this issue and **each time after** pushing code to Github, because the pre-commit bot may add commits to your PRs upstream.

**Comment by DachiCharkviani** [comment]
Hello, I am a first time contributor too, needing to contribute to an open source code for my college assignment. Would appreciate the assignment.

**Comment by cdrini** [comment]
Howdy @Craig-Rosario , go for it; assigning it to you. Please let us know if you have any questions.

I'll keep you in mind for another one @DachiCharkviani  :+1:

**Comment by DachiCharkviani** [comment]
@cdrini Thanks! I also commented on #10095. I would gladly work on that one. I added more information there in the comment of that issue.

**Comment by Craig-Rosario** [comment]
@cdrini Hello, I looked into the issue, the alignment as well as the clipping.
I set up the project locally, and found the solution to the issue, however, when I attempted to change the css properties, they changes don't reflect on the website. I also attempted to make the same changes by inspecting in the browser and could fix it.

I’ve ruled out caching as a potential cause. Additionally, the local build process completes without errors.

I’d appreciate your insights. Let me know if you need more details.
Thanks.

**Comment by Deathnote45** [comment]
hi, Can I contribute on this issue? @cdrini

**Comment by Craig-Rosario** [comment]
@cdrini I fixed the issue and opened a PR for the arrow alignment. Regarding the clipping issue, I'm not entirely sure how you’d prefer it to be handled. Should the list item names not be clipped?

**Comment by cdrini** [comment]
I responded on the PR; thank you @Craig-Rosario !

**Comment by Craig-Rosario** [comment]
Yeah no problem @cdrini , I'll make the necessary changes in the PR. Also yes I solved the issue, no worries now.

**Comment by cdrini** [comment]
Closed by https://github.com/internetarchive/openlibrary/pull/10178 .
